recipe/meta.yaml

```diff
@@ -1,27 +1,25 @@
 {% set name = "PymageJ" %}
-{% set version = "0.1" %}
-{% set sha256 = "a18103bbb773b88542fd84f7dffd36d7310d9644ac44e9bc6f2d5b049aa52f33" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "e0b4e8266423e5d8648920219985bdd59d6f22a7e89bc34f0b40a473399aba3e" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.zip
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
This seemed to work ok for me when plugging it into the browser. Giving it a try here.

Reply:
Looks like we needed a lower case `j` in `name` as well. Making that change fixed this issue